packages/site/src/layout/NavMenu.module.css

```diff
@@ -0,0 +1,60 @@
+
+.navMenuSticky {
+  background-color: var(--color-surface--background);
+  box-sizing: border-box;
+  display: flex;
+  flex-direction: column;
+  gap: var(--space-large);
+  padding: 36px var(--space-base) var(--space-large) var(--space-base);
+  position: fixed;
```

Comment:
This seems a bit overcomplicated, couldn't we have a single container using flex to achieve this? Then we can avoid needing to set the 188px, using `fixed`, using z-index, doing the height calc on the menu container?

Reply:
I updated with your suggestion, let me know if you like it